ui/react-components/components/ToggleButton/ToggleButton.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from "react";
+import { injectIntl } from "react-intl";
+import PropTypes from 'prop-types';
+
+const ToggleButton = (props) => {
+    const {label, disabled} = props;
+    return (
+            <label className={"toggle-btn-switch"}>
+                <input type="checkbox" disabled={disabled}/>
+                <span className={"toggle-btn-slider"} />
+                {label}
```

Comment:
Span is self closed here. Label must be wrapped in span I guess. Please verify this.

Reply:
I have removed label prop for now due to the css changes I have made.